forms-web-app/src/views/appellant-submission/your-details.njk

```diff
@@ -2,16 +2,25 @@
 
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 
 {% block pageTitle %}
   Your details | Appeal a householder planning decision
 {% endblock %}
 
 {% block content %}
+
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorSummary
+    }) }}
+  {% endif %}
+
   <div class="govuk-grid-row">
 
     <div class="govuk-grid-column-two-thirds">
-      <form action="" method="post" novalidate>
+      <form action="/appellant-submission/your-details" method="POST" novalidate>
```

Comment:
not saying it's right or wrong btw, just that leaving `form action=""` means the form will always post back to the current route - which is handy when routes change like they have in this PR / during dev. That's why I've been leaving them as empty. If you'd prefer them explicit, I'm happy to conform.

Reply:
No you're right, better without